scripts/time-filter-fast.js

```diff
@@ -0,0 +1,116 @@
+import {
+  textSummary
+} from 'https://jslib.k6.io/k6-summary/0.0.3/index.js'
+import http from 'k6/http'
+import {
+  check,
+  sleep
+} from 'k6'
+import {
+  generateRandomCoordinate,
+  destinations,
+  timeFilterScenarios as scenarios,
+  setThresholdsForScenarios,
+  countries,
+  summaryTrendStats,
+  deleteTimeFilterMetrics,
+  reportPerDestination
+} from './common.js'
+
+export const options = {
+  scenarios,
+  summaryTrendStats,
+
+  thresholds: {
+    // Intentionally empty. I'll define bogus thresholds (to generate the sub-metrics) below.
+  }
+}
+
+setThresholdsForScenarios(options)
+
+export default function () {
+  const appId = __ENV.APP_ID
+  const apiKey = __ENV.API_KEY
+  const host = __ENV.HOST || 'api.traveltimeapp.com'
+  const countryCode = __ENV.COUNTRY || 'gb'
```

Comment:
time-filter-fast and time-map-fast use protoCountries

Reply:
fixed